README.md

```diff
@@ -1476,6 +1476,28 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='generic-extensions'></a>(<a href='#generic-extensions'>link</a>) When extending generic types, prefer using generic bracket syntax (or sugared syntax for applicable standard library types) instead of generic type constraints. [![SwiftFormat: genericExtensions](https://img.shields.io/badge/SwiftFormat-genericExtensions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#genericExtensions)
```

Comment:
This is always possible when extending fully-bound generic types (e.g. only using `== Foo` constraints and not `: FooProtocol). That's how the SE proposal refers to these ("bound generic types"). Do you think it would be more clear to change this to "When extending bound generic types" instead?

Reply:
> Do you think it would be more clear to change this to "When extending bound generic types" instead?

I like this.